clevercsv/wrappers.py

```diff
@@ -178,7 +177,7 @@ def csv2df(filename, *args, **kwargs):
         warnings.filterwarnings(
             "ignore",
             message="^Conflicting values for .*",
-            category=ParserWarning,
+            category=pd.errors.ParserWarning,
```

Comment:
When Pandas is not installed, pd is None, right? So `pd.errors` should throw an exception?
Sometimes for cases like this I make mock objects with minimal attributes needed for the code not to throw errors.

Reply:
Thanks for taking a look!

Actually if pandas is not installed ``import_optional_dependency("pandas")`` will raise an ImportError, so this is not an issue.